Makefile

```diff
@@ -53,10 +57,10 @@ export HOST_UID
 # i.e. works on most cases. For everything else perhaps more useful to upload a script and execute that.
 shell:
 ifeq ($(CMD_ARGUMENTS),)
-	# no command is given, default to shell
```

Comment:
Comments inside targets need not to have an indent else Windows does this:

```
# no command is given, default to shell
make (e=2): The system cannot find the file specified.
make: *** [Makefile:56: shell] Error 2
```

which is rather confusing, but it's trying to parse the comment as something.